hosted_control_planes/hosted-control-planes-release-notes.adoc

```diff
@@ -150,6 +150,16 @@ $ oc get agents -A
 
 * When you use the console or API to specify an IPv6 address for the `spec.services.servicePublishingStrategy.nodePort.address` field of a hosted cluster, a full IPv6 address with 8 hextets is required. For example, instead of specifying `2620:52:0:1306::30`, you need to specify `2620:52:0:1306:0:0:0:30`.
 
+* On version 4.19.3 and earlier, if you create a hosted cluster on the bare-metal Agent platform and you use the MetalLB Operator, the Cluster Network Operator in the hosted cluster fails to start. The following error occurs:
```

Comment:
Is bare metal an adjective or a noun here (I ask because I'm not sure what "Agent platform" is) . All other instances in our docs are "bare metal Agent" without the hyphen.